content/en/docs/faq/telemetry.md

```diff
@@ -25,7 +25,42 @@ Telemetry collects aggregate information about the size of KubeSphere clusters i
 
 ## Disable Telemetry
 
-Telemetry is enabled by default. To disable Telemetry, follow the steps below after you install KubeSphere.
+Telemetry is enabled by default when you install KubeSphere, while you also have the option to disable it either before or after the installation.
+
+### Disable Telemetry before Installation
+
+When you install KubeSphere on existing Kubernetes clusters, you need to download the file [cluster-configuration.yaml](https://raw.githubusercontent.com/kubesphere/ks-installer/v3.0.0/deploy/cluster-configuration.yaml) for cluster setting. If you want to disable Telemetry, do not use `kubectl apply -f` directly for this file.
+
+{{< notice note >}} 
+
+If you install KubeSphere on Linux, see [Disable Telemetry after Installation](../telemetry/#disable-telemetry-after-installation) directly.
```

Comment:
There should be a prompt for disabling telemetry before installing KubeSphere on Linux machines, I assume it can also be configured in the [config-sample.yaml](https://github.com/kubesphere/kubekey/blob/master/docs/config-example.md).

Reply:
Already confirmed with @Forest-L ; you cannot disable Telemetry before you install KubeSphere on Linux. Users can only disable it after installation if KubeSphere is installed on Linux.

Reply:
@Forest-L  Disabling Telemetry before installation is supposed to be added in Kubekey, please add it in the v3.1.